### DISCUSSION TITLE
[BUG]After Squash and Merge, the forked repository will still remains commits ahead.

### DISCUSSION THREAD

**Discussion by qhy040404** [community/community#16029] [Pull Requests]
```Squash and merge``` will not renew forked repository's status. 
[qhy040404/HowToLiveLonger](https://github.com/qhy040404/HowToLiveLonger)
The commits ahead in this fork repository have been merged in this PR geekan/HowToLiveLonger#104 , but the repo still prompts there are commits ahead

**Comment by MathiasReker** [comment]
Hello. You can report the bug here: https://support.github.com/contact/bug-report

**Reply by qhy040404** [reply]
I'm not sure if this bug still exists.
It seems that this bug report is new feature that I haven't seen before.
